lib/core/toolrepos/step.sh

```diff
@@ -84,11 +105,12 @@ function ih::setup::core.toolrepos::set-auto-update-repositories-job() {
   PLIST_FILE="com.includedhealth.auto-update-repositories"
   LAUNCH_AGENTS_PATH="${HOME}/Library/LaunchAgents/${PLIST_FILE}.plist"
 
+  # shellcheck disable=SC2001
```

Comment:
The updated `shellcheck` linter has a suggestion here but I don't understand if it should be accepted:
```
  GR_HOME_ESC=$(echo "$GR_HOME" | sed 's_/_\\/_g')
                ^-- SC2001 (style): See if you can use ${variable//search/replace} instead.

For more information:
  https://www.shellcheck.net/wiki/SC2001 -- See if you can use ${variable//se...

```